admin_guide/diagnostics_tool.adoc

```diff
@@ -115,3 +115,55 @@ A client with *cluster-admin* access available (for any user, but only the
 current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
+
+[[additional-cluster-health-checks]]
+== Additional Diagnostic Checks
+
+Some additional diagnostic checks are available through the https://hub.docker.com/r/openshift/openshift-ansible[openshift-ansible]
+Docker image. These checks are a set of diagnostic tasks meant to be run against a running OpenShift cluster.
+They are used to ensure Etcd cluster storage sanity, that correct package versions exist on the host machine, and to ensure a working logging stack deployment.
```

Comment:
Not sure I'd try to summarize when we have the full table below. Maybe an even broader summary: "They can report common OpenShift-specific problems for the deployment."